### ISSUE TITLE
Not able to write in another excel file via xlwings

### ISSUE THREAD

**Issue by spbshaan** [ZoomerAnalytics/xlwings#668]
Hi,
I will explain my flow of program and the issue.
I'm copying some input data to 1st excel file to 2nd excel file (I' did not want the excel file to be open when running code & so cannot call from the same). Then, I'm running python code from 1st excel VBA using xlwings. The code runs perfectly. It is suppose to write some data in 2nd excel file. But does not write it.
If I run the code externally from python console, it writes correctly with no issues.
I'm using 0.10.4 of xlwings and it is imported in 1st Excel vba module. I'm opening , saving and closing 2nd file before kicking in the python code. There are no errors in log generated.
I'm not really sure if this is an issue or an individual problem?

**Comment by fzumstein** [comment]
can you attach a self contained miniminal example so I can replicate the issue?

**Comment by spbshaan** [comment]
Hi felix,
Thanks for the quick amazing response! Here is a small version of code.I have used openpyxl for the python part!

VBA part `Set wb1 = ActiveWorkbook
    Set wb2 = Workbooks.Open("C:\Users\Public\Documents\Python Scripts\excelfile2.xlsx")
    Set ws1 = wb1.Worksheets("Input")
    Set ws2 = wb2.Worksheets("InputStatus")
    lrow = ws1.Range("A2").End(xlDown).Row
    ws2.Range("A2:A" & lrow).Value = ws1.Range("A2:A" & lrow).Value
    wb2.Save
    wb2.Close
    Set wb2 = Nothing
    RunPython ("import pythonscriptName; pythonscriptName")`

Python part `from selenium import webdriver
import openpyxl

driver.get("https:website")
**AUNTHENTICATION process here**
xfile = openpyxl.load_workbook('C:\\Users\\Public\\Documents\\Python Scripts\\excelfile2.xlsx')
sheet1 = xfile.get_sheet_by_name('InputStatus')
r3 = sheet1.max_row + 1
for k in range (2,r3):
    reqinput = (sheet1.cell(row=k,column=1).value)	
	requestnbr = driver.find_element_by_name("ReqNbr")
    requestnbr.send_keys(requestnbr)
    driver.find_element_by_name("searchReq").click()
    reqst = driver.find_element_by_xpath("//*[@id='scResponse']").text
	sheet1.cell(row=k,column=2).value = (reqst)
	xfile.save('excelfile2.xlsx')
driver.close()
driver.quit()
`

P.S. I was able to get it running using calling batch file and that works just great. But my ultimate goal is to create exe with frozenpython.

**Comment by spbshaan** [comment]
Hi Felix,

I tested on a simple version of my code. It did not work, Am I doing something wrong here?
Following is the entire code that I tested. VBA calls this python script using this command
RunPython ("import testcode2; testcode2")
The code runs and saves but does not write anything in the file. If i manually kick the py script, it writes and saves. What I have done apart from this is only added the xlwings.bas file in vba module. Was I suppose to add anything else where? anything in py script? 


`import openpyxl

xfile = openpyxl.load_workbook('C:\\Users\\Public\\Documents\\Python Scripts\\testcode2.xlsx')
sheet4 = xfile.get_sheet_by_name('Input')
sheet4.cell(row=2,column=2).value = ("Hello")
sheet4.cell(row=3,column=2).value = ("World?")
sheet4.cell(row=4,column=2).value = ("H R U")
sheet4.cell(row=1,column=2).value = ("Some_Test")
xfile.save('testcode2.xlsx')`
